### DISCUSSION TITLE
Just a beginner

### DISCUSSION THREAD

**Discussion by aalexander47** [community/community#70788] [New to GitHub] [answered]
hello everyone i am @aalexander47 i recently started getting to know about the github and started to upload my academic and personal projects .Can any1 help me to setup up profile as a attractive github profile .....

**Comment by seyLu** [comment]
#### Hi @aalexander47,

You can refer to [awesome-github-profile-readme](https://github.com/abhisheknaiidu/awesome-github-profile-readme) for inspiration. If you have a knack for design and frontend, then a website portfolio would make more sense. As for me, I lean more towards backend so I don't have a website portfolio -- however, I do have projects that either have a demo on YouTube or deployed on free hosting platforms.

**Comment by Isuru-Eranda** [comment, marked as answer]
Watch this YouTube video - https://youtu.be/7FHiew0_NLQ

**Comment by queenofcorgis** [comment]
👋 Welcome to the GitHub Community on Discussions, @aalexander47  Happy to have you here 🙂

To help keep our space clutter-free, we recommend introducing yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204) and checking out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776). This post will be removed within 24-48 hours to keep our community nice and tidy 🧹.
